tests/convection_box_2D/screen-output.txt

```diff
@@ -0,0 +1,189 @@
+
+*** Timestep 1054:  t=0.492509 seconds, dt=0.000450269 seconds
+   Solving temperature system... 0 iterations.
+   Solving Stokes system... 1+0 iterations.
```

Comment:
This and the other files you are adding are for existing tests. What is the reason why you want to add these output files?

Reply:
For the test case I understood that I have to add the output files to the tests directory. Should I do other things?